gallery/workbook/businesscontinuity-insights.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
+	"version": "TemplateGallery/1.0",
+	"categories": [
+		{
+			"id": "Azure Backup",
+			"name": "Azure Backup",
+			"templates": [
+				{
+					"id": "Workbooks/Azure Backup/Backup Report",
+					"author": "Microsoft",
+					"description": "Gain visibility into backup job statistics, consumption trends and optimization opportunities across your Recovery Services vaults and Backup vaults",
+					"name": "Backup Reports (consolidated)(Azure Preview)",
+                                        "isPreview": true
+				},
+				{
+					"id": "Workbooks/Azure Backup/Backup Configuration Status",
+					"author": "Microsoft",
+					"description": "View virutal machines that have not been configured for backup",
+					"name": "Backup Configuration Status(Azure Preview)",
```

Comment:
nit: Can you please add a space between Status and (Azure Preview), same for the rest of the templates